blackbox/docs/sql/reference/alter_user.txt

```diff
@@ -0,0 +1,46 @@
+.. highlight:: psql
+.. _ref-alter-user:
+
+==============
+``ALTER USER``
+==============
+
+Alter an existing database user.
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+Synopsis
+========
+
+::
+
+    ALTER USER
```

Comment:
```
cr> alter user foobar set password = 'nope';
SQLActionException[SQLParseException: line 1:7: no viable alternative at input 'alter user']
cr> alter user foobar set (password = 'nope');
SQLActionException[SQLParseException: line 1:7: no viable alternative at input 'alter user']
```

Reply:
oh, how did it get here @mikethebeer ?

Reply:
@mfussenegger @chaudum my bad. I added documentation since in the beginning we thought `ALTER USER` will be also part of acceptance. Is it okay if I remove it here and add this feature as a separate PR?

Reply:
yes, we should do a completely new PR for ALTER USER